docs/EN/index.md

```diff
@@ -51,96 +51,112 @@ You can also provide support in online support groups by helping users adjust th
 :caption: Overview
 :maxdepth: 2
 
-./Getting-Started/Overview.md
-./Getting-Started/iphone.md
-./Getting-Started/pump.md
-./settings/devices/cgm.md
+Introduction <./Getting-Started/Overview.md>
+
+App Overview <./operate/interface.md>
+
+New User Guide <./Setup/New-User-Setup.md>
+
+Coming from Other AID Systems <./Configuration/transition-qa.md>
+
+Common Issues <./operate/troubleshoot.md>
 
 ```
-<!---
 
-(index-Analyze)=
+(index-Compatibility)=
 
 ```{toctree}
-:caption: Analyze
+:caption: Device Compatibility
 
-./Analyze/EvaluateBasal.md
-./Analyze/EvaluateCR.md
+iPhones & Watches <./Getting-Started/iphone.md>
+
+Pumps <./Getting-Started/pump.md>
+
+CGMs <./settings/devices/cgm.md>
 
 ```
---->
-(index-Configuration)=
+
+(index-Build)=
 
 ```{toctree}
-:caption: Device Configuration
+:caption: Build/Update Trio
 
-./Configuration/Configure.md
-./Configuration/Devices.md
-./Configuration/transition-qa.md
+Build Instructions <./operate/build.md>
 
 ```
+
 (index-Settings-Basics)=
 
 ```{toctree}
-:caption: Basic Settings
-
-./settings/configuration/autotune.md
-./settings/configuration/basalprofile.md
-./settings/configuration/carbratios.md
-./settings/configuration/insulinsensitivites.md
-./settings/configuration/pumpsettings.md
-./settings/configuration/targetglucose.md
-./settings/devices/cgm.md
-./settings/devices/pump.md
-./settings/services/applehealth.md
-./settings/services/fatprotein.md
-./settings/services/nightscout.md
-./settings/services/notifications.md
+:caption: User Settings (all users)
+
+Trio Main Preferences <./settings/configuration/preferences/trio.md>
+
+Pump, CGM, Watch <./Configuration/Devices.md>
+
+Additional Pump Settings <./settings/configuration/pumpsettings.md>
+
+Basal <./settings/configuration/basalprofile.md>
+
+CR <./settings/configuration/carbratios.md>
+
+ISF <./settings/configuration/insulinsensitivities.md>
+
+Set Your Target BG <./settings/configuration/targetglucose.md>
```

Comment:
I'd prefer to keep this `Target Glucose` and remove the `Set Your` part since the others aren't `Set Your ISF`. Bit of a pet peeve that OS-AID systems use sgv rather than bg but glucose covers both.